Thorium-Sim/thorium-nova
# Flight Director Interstellar Starmap

## Description

A starmap Flight Director core that shows the systems and ships on an interstellar level 

## Related Issue








Closes #284
Closes #286 

## How do you know the changes work correctly?






Manual testing:
- Create a new flight
- Assign the client to the flight director station
- Pull up the Starmap Core
- You should see the solar systems on the starmap and ship flying around randomly - that's the player ship
- Clicking the 'Spawn Ship' button at the top should create more ships that also fly around randomly.


## Types of changes



      change)

## Checklist:
